src/main/java/frc/robot/subsystems/PIDSubsystem.java

```diff
@@ -2,5 +2,7 @@
 
 public interface PIDSubsystem extends MovableSubsystem {
     void setSetpoint(double setpoint);
+
+    boolean atSetpoint();
```

Comment:
PIDF subsystem, and make a default for this function that uses getMotor

Reply:
what